test/unit/ngsiv2/examples/contextRequests/updateContextAliasPlugin1.json

```diff
@@ -1,10 +1,19 @@
 {
```

Comment:
Indent style seems to be broken in this file... Please check.

Reply:
Fixed 7ba9d12